### PR TITLE
Add PHYS1181.01_Fall_2021（添加课程2021秋季普通物理①）

### PR DESCRIPTION
**Pull request by arnoliudaxia**
主要是赵爱迪的平行班

- 自己选了一张封面
- 添加了讨论课和课件
- 可能不全但都是真实的

### CODE REVIEW

**Comment by why-in-Shanghaitech** [comment]
非常感谢您的贡献！很高兴iTechX能拥有第一门PHYS课程。我会在近期查看，如有疑问会继续联系您。
- [x] review the pull request
- [x] add to contributors

File: courses/PHYS1181/meta.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            "season": "Fall",
+            "year": "2021",
+            "course_id": "PHYS1181.01",
+            "department": "物质科学与技术学院",
+            "credit": "3",
+            "hours": "48",
+            "prerequisite": "最好有微积分基础。",
+            "evaluation": "未知",
+            "teacher": [
+                "赵爱迪"
```
Comment (why-in-Shanghaitech) [inline courses/PHYS1181/meta.json]: 这里需要先添加人物，然后写人物ID，现在显示异常。我会来修复这个问题。

**Review by why-in-Shanghaitech — COMMENTED** [review]
I do not have push access to your repo, so I need to merge via command line. It might take a while.

**Comment by why-in-Shanghaitech** [comment]
再次感谢您的贡献！内容已在首页可见。

**Comment by arnoliudaxia** [comment]
Thks for your work! Actually I made this two years ago, but It stayed still util I found it in recent days. lol. iTechX is a good place for the openess of info in Shanghai Tech. It built an atmosphere of the Internet in its original purpose. I heared it from a TA in during my freshman year so it's a common wish of us to make iTechX a good place for everyone in this school. Respect to all of you!

**Comment by arnoliudaxia** [comment]
极客万岁!
Long live geeks!
Vive les geeks!

**Pull request merged**